# slow but ok
#diag_elements_gamma: [50, 50, 5, 5, 5]
#diag_elements_lambda: [10, 10, 10, 10, 10]
#diag_elements_b: [50, 50]

diag_elements_gamma: [100, 100, 10, 10, 10]
diag_elements_lambda: [4, 4, 4, 4, 4]
diag_elements_b: [100, 100]

velocity_virtual_threshold: 100.0
los_distance_threshold: 4.0
speed_min: 0.0
speed_max: 4.0
steer_min: -0.52
steer_max: 0.52
k_p_speed: 1.0 #15.0
k_i_speed: 0.01 #0.05
k_p_steer: 0.5
vehicle_length: 0.4
